Localization/es-419.lproj/Localizable.strings

```diff
@@ -22,7 +22,7 @@
 "No Results for “%@“" = "No Results for “%@“";
 "Ok" = "Ok";
 "Permit #" = "Permit #";
-"Please, input name or licence" = "Please, input name or licence";
+"Please, input name or licence" = "Please, input name or license";
```

Comment:
Better to leave both sides the same until the localisation is done (that way your shell incantation will find that the phrase hasn't been localised yet)

Reply:
That's just a mispelling in the English Translation, will submit a fix.

As for the missing phrases, I'll add them in, but also look into why "Boardings" isn't translated - it should be "Inspections"......

Reply:
submitted a fix for this, now both sides (and the code) say "license".